block_carousel.php

```diff
@@ -135,9 +144,10 @@ public function get_content() {
                 $html .= html_writer::start_tag('object');
             }
             $show = ($c == 0) ? 'block' : 'none';
+            $html .= html_writer::start_tag('div', array('style' => "max-width: $width; margin: auto"));
```

Comment:
The height of slidewrap div changes depends on its container, it may exceed the configured height value. I need to add a container and set max-width to limit the height of slidewrap div.